lib/python/pathling/context.py

```diff
@@ -39,7 +39,12 @@ def create(cls, spark: Optional[SparkSession] = None,
                max_nesting_level: Optional[int] = None,
                enable_extensions: Optional[bool] = None,
                enabled_open_types: Optional[Sequence[str]] = None,
-               terminology_server_url: Optional[str] = None) -> "PathlingContext":
+               terminology_server_url: Optional[str] = None,
+               token_endpoint: Optional[str] = None,
+               client_id: Optional[str] = None,
+               client_secret: Optional[str] = None,
+               scope: Optional[str] = None,
+               token_expiry_tolerance: int = 120) -> "PathlingContext":
```

Comment:
It might be more consistent to make it:
```token_expiry_tolerance: Optional[int] =None ```
and rely on the default from Java code.

Reply:
Fixed in fb3da7de620ed6f126e07d284893935bb5e5a424